encode/src/main/scala/org/broadinstitute/monster/etl/encode/EncodeIngest.scala

```diff
@@ -66,9 +66,54 @@ object EncodeIngest {
     // Add processing steps between the read and write here.
     pipelineContext
       .jsonFile[JsonObject](parsedArgs.experimentsJson)
+      .map { jObj =>
+        jObj.filterKeys(Filters.EncodeFields.contains)
+      }
+      .map { jObj =>
+        Filters.RenamedEncodeFields.foldLeft(jObj) {
+          case (currentJObj, (oldFieldName, newFieldNames)) =>
+            currentJObj(oldFieldName).fold(currentJObj) { value =>
+              newFieldNames.foldLeft(currentJObj) { (currentJObjModify, newFieldName) =>
+                currentJObjModify
+                  .add(newFieldName, value)
+                  .remove(oldFieldName)
+              }
+            }
+        }
+      }
       .saveAsJsonFile(parsedArgs.outputDir)
 
     pipelineContext.close().waitUntilDone()
     ()
   }
 }
+
+object Filters {
+
+  val EncodeFields =
```

Comment:
This name will turn out to be too generic; `ExperimentEncodeFields` would be more accurate.